jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -818,12 +818,10 @@ ecma_uint32_to_utf8_string (uint32_t value, /**< value to convert */
  *
  * @return number - result of conversion.
  */
-ecma_number_t
+ecma_number_t __attr_always_inline___
```

Comment:
Is this needed? Where does it make a difference? Performance results show no significant change that would justify the need for this attribute. (BTW, does it have an effect on the code size?)